src/cares_wrap.cc

```diff
@@ -1399,24 +1384,27 @@ static void Initialize(Local<Object> target,
   target->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "AI_V4MAPPED"),
               Integer::New(env->isolate(), AI_V4MAPPED));
 
+  auto isConstructCall = [](const FunctionCallbackInfo<Value>& args) {
+        CHECK(args.IsConstructCall());
```

Comment:
Only two spaces of indent and call the variable `is_construct_call` (or `is_construct_call_callback` to indicate it's a function pointer.)